apps/ima/sections/summary-data-sections.js

```diff
@@ -339,6 +339,11 @@ module.exports = {
               if (field.field === 'family-member-full-name') {
                 field.isAggregatorTitle = true;
               }
+              if (field.field === 'reference-number-option') {
+                if(field.value.includes('unique-application-number') && (field.value.includes('home-office-reference-number'))) {
+                  field.parsed = 'Unique Application Number and Home Office Reference Number';
+                }
+              }
```

Comment:
Would be ideal to add in a few more checks after this one, to improve the wording of the other options, like this:

```
else if(field.value === 'unique-application-number') {
  field.parsed = 'Unique Application Number';
} else if(field.value === 'home-office-reference-number') {
  field.parsed = 'Home Office Reference Number';
} else {
  field.parsed = 'Do not know';
}
```

Reply:
Changes done